geometry_msgs/Point pose
string image_base64